packages/amplify-category-function/src/provider-utils/awscloudformation/utils/lambda-layer-cloudformation-template.ts

```diff
@@ -76,20 +29,81 @@ export default function generateLayerCfnObj(parameters) {
           Ref: 'AWS::Region',
         },
       },
-      Permission: {
-        Value: {
-          Ref: 'LambdaLayerPermission',
-        },
-      },
     },
   };
+  const POLICY_RETAIN = DeletionPolicy.Retain;
+  const layer = new Lambda.LayerVersion({
+    CompatibleRuntimes: parameters.runtimes.map(runtime => runtime.cloudTemplateValue),
+    Content: {
+      S3Bucket: '<S3Bucket>',
```

Comment:
This was changed in my PR to use refs (see above diff)

```js
    Content: {
      S3Bucket: Fn.Ref('deploymentBucketName'),
      S3Key: Fn.Ref('s3Key'),
    },
```

Reply:
done